languages/kotlin/concepts/basics/about.md

```diff
@@ -1 +1,41 @@
-TODO: add information on basics concept
+# What's next?
+
+## Type inference
+
+Kotlin compiler can infer types for functions and variables in most of the cases. However, declaring function return types explicitly for **public API** is a good practice.
+
+## Entry point
+
+To run your Kotlin program you need to define so-called entry point: function with name `main` with or without arguments. However, in Exercism course we are using automatic tests that are using other functions defined in exercises.
+
+Entry point with arguments (you can use them for building CLI applications):
+
+```kotlin
+fun main(args: Array<String>) {
+    println("Hello, Exercism!")
+    println("Program args are: $args")
+}
+```
+
+Or ignore arguments completely:
+
+```kotlin
+fun main() {
+    println("Hello, Exercism!")
+}
+```
+
+## `Unit` return type
+
+You've seen that some functions (like `main()` above) are not returning value. However, they are implicitly returning value that is called `Unit` (quite similar to `void` in Java/C/C++):
+
+```kotlin
+fun run() {}
+
+// is the same as
+fun run(): Unit { return Unit }
+```
+
+Returning `Unit` (and using it in function declaration) is completely optional and is omitted in most of the cases.
+
+[intellij-idea-ic]: https://www.jetbrains.com/idea/download/
```

Comment:
Did I get it right that we need to extract links like that into `links.json`.

Reply:
Probably. Not every link that is in the `about.md` document needs to be extracted to the `links.json` file. For more information, see https://github.com/exercism/v3/issues/2499.